### PR TITLE
Feature/activity drawer

### PR DESCRIPTION
**Pull request by WRadoslaw**
- #3155

### CODE REVIEW

**Comment by dmtrjsg** [comment]
to be reviewed as the last one

**Comment by WRadoslaw** [comment]
@thesan Ready for review

File: packages/ui/src/app/pages/Forum/Forum.tsx [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+  const [value, toggle] = useToggle(false)
   return (
     <PageLayout
       header={
-        <ForumPageHeader title={<PageTitle>Forum</PageTitle>}>
+        <ForumPageHeader
+          title={<PageTitle>Forum</PageTitle>}
+          buttons={
+            <ButtonGhost size="medium" onClick={toggle}>
+              Test
```
Comment (thesan) [inline packages/ui/src/app/pages/Forum/Forum.tsx:22]:
```suggestion
              Forum Activities
```

**Review by thesan — CHANGES_REQUESTED** [review]
Just a copy change

**Review by thesan — APPROVED** [review]
All good ! Just one more PR left to squash it all on `dev` :rocket: 
Sorry it looks like forgot about this one last Thursday :sweat:

**Pull request merged**